independent-publisher-connectors/Notion/apiProperties.json

```diff
@@ -0,0 +1,23 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#F6E8EB",
```

Comment:
@sekharmalla Please update it to the default iconBrandColor of all the independent publisher connectors.

`"iconBrandColor": "#da3b01"`

Reply:
successfully updated.